tests/test_bar_data.py

```diff
@@ -382,8 +371,12 @@ def test_spot_price_is_unadjusted(self):
     def test_spot_price_is_adjusted_if_needed(self):
         # on cls.days[1], the first 9 minutes of ILLIQUID_SPLIT_ASSET are
         # missing. let's get them.
-        day0_minutes = self.env.market_minutes_for_day(self.days[0])
```

Comment:
Another thing we could have done here would be to set `self.days` in the testcase's `init_{instance,class}_fixtures`.  That way the tests still get a nice variable name, but we don't clutter the global fixture namespace.  Probably not worth changing now though.